src/vs/workbench/parts/extensions/common/extensions.ts

```diff
@@ -74,6 +74,7 @@ export const IExtensionsWorkbenchService = createDecorator<IExtensionsWorkbenchS
 export interface IExtensionsWorkbenchService {
 	_serviceBrand: any;
 	onChange: Event<void>;
+	onRecommendationChange: Event<void>;
```

Comment:
`ExtensionsWorkbenchService` is the place for install/update/disable/uninstall/query extensions.
All recommendation related work is happening in `ExtensionsTipsService`. 
So how about moving both the `onRecommendationChange` event and the `ignore` function to `ExtensionsTipsService` ?

Also rename `ignore` to `ignoreExtensionRecommendation`